README.md

```diff
@@ -10,6 +10,14 @@ This repository guides developers wanting to contribute to Scroll. You can start
 
 Thanks for your interest in contributing to Scroll. Check out the [CONTRIBUTING](https://github.com/scroll-tech/contribute/blob/main/CONTRIBUTING.md) page for a guide on how to contribute.
 
+### GitPOAP
```

Comment:
Would suggest updating the CONTRIBUTING.md with this instead. [Here](https://github.com/scroll-tech/contribute-to-scroll/blob/main/CONTRIBUTING.md?plain=1#L42)